harvard-lts/mirador-help-plugin
# Address redundant ul element in Help modal window

**Address redundant ul element in Help modal window**
* * *

**JIRA Ticket**: [LTSVIEWER-244](https://jira.huit.harvard.edu/browse/LTSVIEWER-244)

# What does this Pull Request do?
Uses the new plugin template and updates the help modal window to be an accessible list of links; also removed "Take a tour" link as it's no longer relevant with the new viewer

# How should this be tested?
* Spot the problem
   * Navigate to the help window on the main branch or qa site
   * If you inspect the links you'll notice they're in an unordered list `<ul>`, but are not coded as nested list items `<li>`s
   * If you have SiteImprove (I use the chrome extension for this), you'll see that the error related to this is "Container element is empty"
* View the fix
   * Spin up this branch and open up the help window
   * Inspect the links again, they should appear as nested list items, `<li>`s
   *  If you have SiteImprove (I use the chrome extension for this), you'll see that the error related to this ("Container element is empty") no longer appears
* Help modal should no longer have a "Take a tour" link

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
